Content(mission): A dead mans path (#7803)
endless-sky/endless-sky: data/human/human missions.txt

			label end
			`	You say your goodbyes, leaving to your ship and returning to the stars.`
				decline

mission "A dead man's path I"
	name "Siblings to New China"
	description "Bring these sibling to New China to hear their fathers last will."
	passengers 2
	source
		near "Merak" 3 6
	destination
		planet "New China"
	to offer
		random < 20
	on offer
		conversation

> 			`As you walk through the spaceport with nothing on your mind, a young woman and man, who are both dressed very formal, approach you. The woman starts talking to you, "Hello, are you a spaceship captain?" You answer, "Indeed, I am." She looks happy, "That is good to hear, my brother and I were searching for someone who could give us a ride to New China. We received notice that our fathers last will will be published there. Do you think you could do this, we will pay for your service of course."`

Author: ok

> 			`As you walk through the spaceport with nothing on your mind, a young woman and man, who are both dressed very formal, approach you. The woman starts talking to you, "Hello, are you a spaceship captain?" You answer, "Indeed, I am." She looks happy, "That is good to hear, my brother and I were searching for someone who could give us a ride to New China. We received notice that our fathers last will will be published there. Do you think you could do this, we will pay for your service of course."`

Author: just tried to add some spice XD, but I guess you are right

> 			`As you walk through the spaceport, a young woman and man, who are both dressed very formally, approach you. The woman begins, "Hello, are you a spaceship captain?" You answer, "Indeed, I am." Her face brightens up with a smile, and she continues, "That is good to hear, my brother and I were searching for someone who could give us a ride to New China. We received notice that our father's last will will be published there. Do you think you could do this? We will pay for your service, of course."`

Reviewer: ```suggestion
			`As you walk through the spaceport, a young woman and man, who are both dressed very formally, approach you. The woman begins, "Hello, are you a spaceship captain?"`
			`	You answer, "Indeed, I am."`
			`	Her face brightens up with a smile, and she continues, "That is good to hear, my brother and I were searching for someone who could give us a ride to New China. We received notice that our father's last will will be published there. Do you think you could do this? We will pay for your service, of course."`
```
it's good to space the text in general and I think it makes sense when smbd else speaks, especially since here you're speaking for the player and choosing they don't lie

Or make this a choice, maybe even a forced one, I think that Amazinite said he didnt like these kind of speak for the player choices

Author: Working on it

Author: fixed, I think

> 			`As you walk through the spaceport with nothing on your mind, a young woman and man, who are both dressed very formally, approach you. The woman begins, "Hello, are you a spaceship captain?" You answer, "Indeed, I am." Her face brightens up with a smile, and she continues, "That is good to hear, my brother and I were searching for someone who could give us a ride to New China. We received notice that our father's last will will be published there. Do you think you could do this? We will pay for your service, of course."`

Author: ```suggestion
			`As you walk through the spaceport, a young woman and man, who are both dressed very formally, approach you. The woman begins, "Hello, are you a spaceship captain?" You answer, "Indeed, I am." Her face brightens up with a smile, and she continues, "That is good to hear, my brother and I were searching for someone who could give us a ride to New China. We received notice that our father's last will will be published there. Do you think you could do this? We will pay for your service, of course."`
```

			choice
				"Of course I can."
					goto "Agree"
				"I am sorry, but I have more urgent things to do."
			`    They both look not very happy as you say that, but they nod and turn away.`
				decline
			label "Agree"

> 			`    "Fantastic," she says, "my name is Sophie. This is my younger brother," she points at the man to her side. "Hi, I am Liam," he says, "we have been looking for a ride for some time now, we already started to worry we wont get there in time," he tells you. You nod in agreement, "A lot of captains are very busy these times, you are lucky that I have some time. My name is <first> <last>." Theynod in agreement and then you lead them to your spaceship.`

Author: outdated

				accept
	on complete
		conversation
			`When you arrive at New China's spaceport, the siblings head off to the place where their fathers last will will be read out. Liam turns to you. "Captain <last>, I have transferred the payment to your account, would it be okay for you to wait here until we come back? We will need a ride home." You agree, and go to the local bar to kill some time.`
			`    After about two hours they come back, both seem somehow excited. You flag them down from your table, and they sit down. Sophie begins talking, "Captain <last>, it turns out we will need a bit more than just a ride home. If you want to hear us out, come here again in two hours because my brother and I need to buy some things."`
		payment 50000

> 

Reviewer: you could also set a log upon completion of each part

Author: Noted

mission "A dead man's path II"
	name "Following Footprints"
	description "Bring Sophie and Liam to Tundra in the Cebalrai System."
	passengers 2
	source
		planet "New China"
	destination
		planet "Prime"
	waypoint "Terminus"
	waypoint "Aspidiske"
	waypoint "Gorvi"
	waypoint "Enif"
	to offer
		has "A dead man's path I: done"
	on offer
		conversation

> 			`They both look very happy when they see you again. "So you have decided to help us," Liam asks. "I do not know yet, maybe you should tell me first what this is about," you answer. "Of course," Sophie says, "my brother is just can not wait to get started. Maybe I should start with who our father was. We do not know much about him. He came home only occasionly, do not get me wrong, our parents were not in conflict, his passion just did not allow him to stay at one place very long. He was a captain, not unlike you. He was restless to explore even the furthest parts of the galaxy and we believe he came further than most human captains. Sometimes he brought small items back home that looked like nothing we know." She pauses for a moment, lost in memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. But because he is a great fan fantasy romans, he said he would not make it easy for us. Instead he left behind a route of waypoints that we should visit before we go to the place where he left something behind. Of course we could go directly to this place, but we want to respect our fathers last will." She looks directly at you now, "And that is why we need your help. Would you help us to get to all waypoints?"`

Reviewer: this is too big of a text wall that needs spacing also but I think that we should not make the player speak without a choice in general; FW is a bad example for that

it's even best to have a fake choice than to have the narrator speak for us especially on things like that

Author: will integrate some suggestion from bene-dictator to smooth this out

Author: fixed

			choice
				"I would be glad to."
					goto "Accept"

> 				"I am sorry but that sounds a bit stupid to me."

Reviewer: stupid may be a bit too rough

Reviewer 2: Far-fetched would be better.

Reviewer 3: Some other alternatives to "a bit stupid":
"overly complicated"
"extravagant"
"silly"
"wasteful"

```suggestion
				"I am sorry but that sounds overly complicated to me."
```

That would my personal impression, but there's lots of valid options.

			scene "outfit/old sword"
			`    You all get closer to the tablet. A short message is written on it:`
			`    My dear children, this is the summary of all the things I have done, enjoy it!`
			`    All of you walk around for some time and look at the various artifacts. You notice the siblings standing in one corner talking about something. They wink you to them. Sophie starts, "Captain <last>, we have thought about this sometime and we think it would be a shame to just keep this sword as a memory. It needs to be used. It was our fathers one and only weapon, which probably means that is has all kind of alien tecnology integrated," she takes the sword and reaches it to you. "It is yours now." You look at her, surprised. "Are you sure about this?" you ask. "Yes we are," Liam says, "please take it on some more adventures." You take the sword.`
			`    "Well, it seems like this is the end of our short journey together, Capatin <last>," Sophie says. "It was a pleasure to travel with you," says Liam. You say goodbye to them and leave them alone in the dome.`

> 		outfit "The old mans sword" 1

Author: ```suggestion
		outfit "Old Man's sword" 1
```

			`    After about two hours they come back, both seem somehow excited. You flag them down from your table, and they sit down. Sophie begins talking, "Captain <last>, it turns out we will need a bit more than just a ride home. If you want to hear us out, come here again in two hours because my brother and I need to buy some things."`
		payment 50000

mission "A dead man's path II"
	name "Following Footprints"
	description "Bring Sophie and Liam to Tundra in the Cebalrai System."
	passengers 2
	source
		planet "New China"
	destination
		planet "Prime"
	waypoint "Terminus"
	waypoint "Aspidiske"
	waypoint "Gorvi"
	waypoint "Enif"
	to offer
		has "A dead man's path I: done"
	on offer
		conversation
			`They both look very happy when they see you again. "So you have decided to help us," Liam asks`
			choice
				"I do not know what you want me to do yet."
				"Sorry, but I would be stupid to accept a mission I do not know anything about."

> 			`"Of course," Sophie says, "my brother is just can not wait to get started. Maybe I should start with who our father was. We do not know much about him. He came home only occasionly, do not get me wrong, our parents were not in conflict, his passion just did not allow him to stay at one place very long. He was a captain, not unlike you. He was restless to explore even the furthest parts of the galaxy and we believe he came further than most human captains. Sometimes he brought small items back home that looked like nothing we know." She pauses for a moment, lost in memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. But because he is a great fan fantasy romans, he said he would not make it easy for us. Instead he left behind a route of waypoints that we should visit before we go to the place where he left something behind. Of course we could go directly to this place, but we want to respect our fathers last will." She looks directly at you now, "And that is why we need your help. Would you help us to get to all waypoints?"`

Reviewer 3: ```suggestion
			`"Of course," Sophie says, "my brother is just can not wait to get started. Maybe I should start with who our father was. We do not know much about him. He came home only occasionally, do not get me wrong, our parents were not in conflict, his passion just did not allow him to stay at one place very long. He was a captain, not unlike you. He was restless to explore even the furthest parts of the galaxy and we believe he travelled further than most human captains. Sometimes he brought small items back home that looked like nothing we know." She pauses for a moment, lost in memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. But because he is a great fan fantasy books, he said he would not make it easy for us. Instead he left behind a route of waypoints that we should visit before we go to the place where he left something behind. Of course we could go directly to this place, but we want to respect our fathers last will." She looks directly at you now, "And that is why we need your help. Would you help us to get to all waypoints?"`
```
This suggestion has a bunch of changes. Most are flow or spelling ones that should be changed, however one, romans->novel is optional depending on how you are intending to portray Sophie.

romans is French for novels. if Sophie speaks French and just happens to not know the word for novel (or it merely slipped into her English dialog, as is not uncommon for vocabulary to cross over), and thus this inclusion in her dialog is portraying a bit of her background, then leaving it in is good and clever. If she's not a french speaker or if this detail is irrelevant, than it should be swapped to novel.

Reviewer 2: Honestly thought it was fantasy Romans...

Author: No french background intended

			label end
			`	You say your goodbyes, leaving to your ship and returning to the stars.`
				decline

mission "A dead man's path I"
	name "Siblings to New China"
	description "Bring these siblings to New China to hear their father's last will."
	passengers 2
	source

> 		near "Merak" 3 6

Reviewer 2: ```suggestion
		near "Merak" 3 6
		not attributes "station"
```
It wouldn't work with one of my suggestions, and also it would make more sense. There is a chance that can be offered on somewhere that wouldn't work like Huginn Station.

	destination
		planet "New China"
	to offer
		random < 20
	on offer
		conversation

> 			`As you walk through the spaceport, a young woman and man, who are both dressed very formally, approach you. The woman begins, "Hello, are you a spaceship captain?"`

Reviewer 2: ```suggestion
			`The spaceport on <origin> is lively tonight. There are porters running to and from depots with crates, crewmen buying rounds of drinks for each other around the bars before they hit the bed, and bands of Navy officers jovially greeting captains and systematically inspecting ships. The energy and self-purpose of this spaceport is on full display this one night.`
			`	The only exception to the bustle and activity are a young woman and man, both dressed very formally, who are seemingly lost and having no luck approaching anybody. You watch they drift from one person to another, each time being sent off without an answer. Finally, they come towards your direction.`
			`	Their faces show clearly that they are on their last legs of hope. It is only with reluctance that the woman says, "Excuse me, are you a spaceship captain?"`
```
This way is a bit more interesting, I reckon.

Author: wow, this is good

			choice
				"Depends on why you are asking."
					goto suspicious

> 				"Indeed, I am."

Reviewer 3: For whatever it's worth, my general rule of thumb is that option 1 is the "accept and proceed through the conversation in the fastest way possible, bypassing explanations or questions where applicable." option. The "player just hammers 1 to get through it fast and onto the action" option, basically. This isn't a rule or even a standard, but it's what I generally try to do to keep it consistent.

			`    Her face brightens up with a smile, and she continues, "That is good to hear, my brother and I were searching for someone who could give us a ride to New China. We received notice that our father's last will will be published there. Do you think you could do this? We will pay for your service, of course."`
				goto helpornot
			label suspicious
			`    She laughs, "No need to be suspicious, my brother and I were just searching for someone who could give us a ride to New China. We received notice that our father's last will will be published there. Do you think you could do this? We will pay for your service, of course.`
			label helpornot
			choice

> 				"Of course I can."
> 					goto "Agree"
> 				"I am sorry, but I have more urgent things to do."
> 			`    Her smile fades away as you say that, but they nod and turn away.`
> 				decline
> 			label "Agree"

Reviewer 2: ```suggestion
			`	"Of course I can."`
					goto agreed
			`	"I'm sorry, but I've got more urgent things to do. You'll need to find someone else."`
			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, with both heads looking down.`
				decline
			label agreed
```

			choice
				"You will be surpised how fast my ship flies, my name is <first> <last> by the way."
				"Yes, a lot of captains are busy these days. Call me Captain <last>."
			`   You lead them to your spaceship.`
				accept
	on complete
		conversation
			`When you arrive at New China's spaceport, the siblings head off to the place where their fathers last will will be read out. Liam turns to you. "Captain <last>, I have transferred the payment to your account, would it be okay for you to wait here until we come back? We will need a ride home." You agree, and go to the local bar to kill some time.`
			`    After about two hours they come back, both seem somehow excited. You flag them down from your table, and they sit down. Sophie begins talking, "Captain <last>, it turns out we will need a bit more than just a ride home. If you want to hear us out, come here again in two hours because my brother and I need to buy some things."`

> 		payment 50000

Reviewer 3: I could see dropping it as far as 30k, but generally speaking I think actual story missions like this should be a more generous than comparable jobs.

> 		payment 50000

Reviewer: Totally agreed on that we need to encourage ppl to find them and not just mass trade

mission "A dead man's path II"
	name "Following Footprints"

> 	description "Bring Sophie and Liam to Tundra in the Cebalrai System."

Reviewer 2: ```suggestion
	description "Bring Sophie and Liam through <waypoints>, before finally bringing them to <destination>."
```

	passengers 2
	source
		planet "New China"
	destination
		planet "Prime"
	waypoint "Terminus"
	waypoint "Aspidiske"
	waypoint "Gorvi"
	waypoint "Enif"
	to offer
		has "A dead man's path I: done"
	on offer
		conversation
			`They both look very happy when they see you again. "So you have decided to help us," Liam asks`
			choice
				"I do not know what you want me to do yet."
				"Sorry, but I would be stupid to accept a mission I do not know anything about."

> 			`"Of course," Sophie says, "my brother is just can not wait to get started. Maybe I should start with who our father was. We do not know much about him. He came home only occasionally, do not get me wrong, our parents were not in conflict, his passion just did not allow him to stay at one place very long. He was a captain, not unlike you. He was restless to explore even the furthest parts of the galaxy and we believe he travelled further than most human captains. Sometimes he brought small items back home that looked like nothing we know." She pauses for a moment, lost in memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. But because he is a great fan of fantasy books, he said he would not make it easy for us. Instead he left behind a route of waypoints that we should visit before we go to the place where he left something behind. Of course we could go directly to this place, but we want to respect our fathers last will." She looks directly at you now, "And that is why we need your help. Would you help us to get to all of the waypoints?"`

Reviewer 2: ```suggestion
			`Sophie looks at her brother amusedly. "Of course, my brother just can't wait to get started," She turns back to you. "Maybe I should start with who our father was. We do not know much about him. He came home only occasionally, do not get me wrong, our parents were not in conflict, his passion just did not allow him to stay at one place very long. He was a captain, not unlike you. He was restless to explore even the furthest parts of the galaxy and we believe he travelled further than most human captains. Sometimes he brought small items back home that looked like nothing we know." She pauses for a moment, lost in memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. But because he is a great fan of fantasy books, he said he would not make it easy for us. Instead he left behind a route of waypoints that we should visit before we go to the place where he left something behind. Of course we could go directly to this place, but we want to respect our fathers last will." She looks directly at you now, "And that is why we need your help. Would you help us to get to all of the waypoints?"`
```
I'm going to improve this tomorrow.

			choice
				"I would be glad to."
					goto "Accept"
				"I am sorry but that sounds overly complicated to me."
			`    "Well, seems like we need to find somebody else," Sophie says. They both shake your hand and walk away.`
				decline
			label "Accept"
			`    Liam pulls a map out of his bag. "These are the systems our father wants us to visit. Apparently they all have something unique to them. We hope for a nice trip."`
				accept
	on enter "Terminus"
		dialog "You all look fascinated at a anomalie in the system."
	on enter "Aspidiske"
		dialog "The beauty of this star stops every conversation for a moment."
	on enter "Gorvi"
		dialog "You chat about how much power a supernova must have, if even an old one leaves this much behind."
	on enter "Enif"

> 		dialog "The strange but impressive unfinsihed Quarg ringworld shows all of you how much the human race has to learn."

Author: Aspidiske is there because I like this type of Star XD

			`    Liam pulls out a map, not a star map but a local map. "Apparently our father bought a whole mountain on this planet. Let's get there."`
			choice
				"Sounds good, lets get there."
					goto shuttle
				"We could also have a look around here, see why your father chose this planet."
			`    "You are right, Sophie says, "lets just relax in this town first."`
			`    The three walk around for some time. At one giant plaza you see one of the newest ship of Betelgeuse being showed off. Liam's face brightens as if he just understood something. "I think our father chose this planet, because he always was a fan of the Betelgeuse ships. If I recall correctly, he once was gifted one of their newest ships for test flighting it." Sophie nods, "Right now that you say, I remember that too. Anyways, shall we get to the mountain now?"`
			choice
				"We should take a shuttle, they are the fastest way."
					goto shuttle
				"We have time, lets take the maglev."
					goto maglev
			label shuttle
			`    After a short shuttle flight, you arrive in front of a single, not too large mountain, that peaks out of the rest of the landscape.`
				goto front-mountain
			label maglev

> 			`    You take the maglev and get a tour through some regions of the planet. Although it is hard to get any detail because of the high speed, you get a good impression. After an hour you end up close to a mountain that stands solely in the landscape.`

Author: Hmmm...

			label front-mountain

> 			`    Liam, with help of the map, guides to a flat wall. He looks a bit embarrassed at you two, "Our father wants us to say: Edr- i fen. I do not know what it means, but I am most certain it refers to one of this fantasy novels he used to read."`

Author: It is elvish for open the door

			choice
				"Sounds funny, lets do it."
				"What a strange man, but I like his sense of humor."
			`    You count to three and then shout out the phrase together. You all have to laugh a bit after that. Suddenly a blue line appears on the wall. The line shapes rectangle. After it has finished the shape, the stone inside the rectangle just disappears. All of you look surprised. "I could have sweared the stone was real," Liam says silently.`
			choice
				"Just like a real adventure, lets go!"
					goto enter
				"We do not know what is inside there, lets be careful."
			`    "Oh, now that you say it, I recently bought a small camera drone," Liam says, "we can use it to scout out the hallway." He takes out a small box with a drone inside. "It is only in infrared, so we wont see much details," he adds. He starts the drone and guides it with a small remote control with integrated display. "I think it is safe, we can go," he tells you after some time.`
			label enter

> 			`    You enter the halway behind the opening. The walls are perfectly smooth and the light seems to come out of nowhere. "To use alien technology for something like this," Sophie laughs, "That is totally our dad." You proceed until you enter dome like room. On the walls all kind of alien artifacts line up next to each other. In the center of the room is a small slope with a strange looking sword inside. A small tablet stands on a socket in front of it.`

Author: Hmmm, yes that sounds reasonable

			choice
				"[get closer to the tablet]"
					goto sword-scene
				"[take a look around first]"
			`    You walk around, looking at the various alien artifacts.`
			`    A lot of the artifacts look similar to Sheragi technology, you ask yourself where he got this stuff.`
				to display
					has "Sheragi Archaeology: The Emerald Sword 3: done"
			`    After some time Sophie calls you both to the tablet.`
			label sword-scene
			scene "scene/sword dome"

> 			`    You all get closer to the tablet. A short message is written on it:`
> 			`    My dear children, this is the summary of all the things I have done, enjoy it!`

Reviewer 2: ```suggestion
			`	Approaching the tablet, there is a short message written on it:`
			`	"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
			`	But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
			`	This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
			`	Your father."`
```
To be honest... I did not really understand the message from their father originally. I've tried to explain it a bit better by running through their father's life, make it a bit more grandiose. I know I said I wouldn't make large changes, sorry :)

Author: No problem, I like the changes

> 			choice
> 				"That is one of the best gifts I have ever seen."
> 				"Now this is some cool stuff."

Reviewer 2: ```suggestion
```
I think best to take this part out. It will be a personal moment for the siblings.

Author: you are right

> 			`    You leave the two of them alone for a bit, walking around and looking at the atrifacts.`
> 			`    After some time they wink you to them. Sophie starts, "Captain <last>, we have thought about this sometime and we think it would be a shame to just keep this sword as a memory. It needs to be used. It was our fathers one and only weapon, which probably means that is has all kind of alien tecnology integrated," she takes the sword and reaches it to you. "It is yours now." You look at her, surprised.`

Reviewer 2: ```suggestion
			`	You leave the two of them alone for a bit, walking around and looking at the artifacts.`
			`	After some time they call you to them. Sophie starts, "Captain <last>, Liam and I have thought through about this, and we think it would be a shame if we just kept this sword as a memory to him. It needs to be used. It was our father's one and only weapon, and it has been many places. Take it to many more." She pulls the sword from the pedestal and hands it to you. "It is yours now."`
```

> 			choice
> 				"This is too much for me to take."
> 					goto decline-offer
> 				"I would be more than grateful to take it."
> 					goto accept-sword
> 			label decline-offer
> 			`    "Are you sure," Liam asks, "this is a one time offer."`

Reviewer 2: ```suggestion
			choice
				`	"I would be more than grateful to accept it."`
						goto accepted
				`	"This is too much for me to take."`
						goto declined
			label declined
			`	"Are you sure?" Liam asks. "This is a one time offer. It would be wasted on us."`
```

> 			choice
> 				"Yes, I am sure."
> 				"Well, if you say it like this, I will take it."
> 					goto accept-sword
> 			`    "I see, at least take this extra payment," Liam transfers 50 thousand credits to your bank account.`
> 			action
> 				payment 50000
> 			`    "It is for you traveling so much with us," Sophie adds.`
> 				goto end

Reviewer 2: ```suggestion
			choice
				`	"Yes, I'm sure."`
				`	"Well, if you frame it that way, I will take it."`
					goto accepted
			`	"I see. Please, at least take this extra payment of <credits>," Liam says. "It is for you traveling so much with us."`
				action
					payment 50000
				goto end
```

> 			label accept-sword
> 			`    "Please take it on some more adventures," Sophie says. You take the sword. It feels strangely light and as you swing it an energy bolts shoots out of it. Better be careful with his.`
> 			action
> 				outfit "Old Man's sword" 1
> 			label end
> 			`    "Well, it seems like this is the end of our short journey together, Capatin <last>," Sophie says. "It was a pleasure to travel with you," says Liam. You say goodbye to them and leave them alone in the dome.`
> 		payment 50000

Reviewer 2: ```suggestion
			label accept-sword
			`	"Please take it on some more adventures," Sophie says.`
			action
				outfit "Old Man's Sword" 1
			label end
			`	"Well, it seems like this is the end of our short journey together, Captain <last>," Sophie says.`
			`	"It was a pleasure to travel with you," adds Liam. "We will always thank you for helping us with our father's will. Here's your payment of <payment>, and farewell." `
			`	You say goodbye to them and leave them alone in the dome.`
				payment 50000
				decline
```
I think the part about the energy bolts should be 'learned' after using it, you did say that you wanted a part of it to be unknown. I also think that a potential 100,000 credits for this mission is massive, although you do bring them around the entirety of human space.

Author: If you think about this, the amount of jumps needed, equals even more than 100,000 but I am open to changes

			`	Their faces show clearly that they are on their last legs of hope. It is only with reluctance that the woman says, "Excuse me, are you a spaceship captain?"`
			choice
				`	"Yes, my name's Captain <last>."`
				`	"Depends on what you are asking."`
					goto suspicious
			`	Her face brightens up with a smile, and she continues, "That's good to hear! My brother and I were searching for someone who could give us a ride to New China, we received notice that our father's last will is going to be published there. Do you think you could bring us there? We will pay you for your service, of course."`
				goto helpornot
			label suspicious
			`    She laughs, "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could bring us there? We will pay you for your service, of course.`
			label helpornot

> 			choice
> 			`	"Of course I can."`
> 					goto agreed
> 			`	"I'm sorry, but I've got more urgent things to do. You'll need to find someone else."`

Reviewer 2: ```suggestion
			choice
				`	"Of course I can."`
						goto agreed
				`	"I'm sorry, but I've got more urgent things to do. You'll need to find someone else."`
```

			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, with both heads looking down.`
				decline
			label agreed
			`    "Fantastic! My name is Sophie, and this is my younger brother," she says, pointing at the man to her side, who goes to shake your hand. "Hi, I'm Liam. We have been looking for a ride for some time now, we've already started worrying that we wouldn't get there in time! You'll have to bring us there quickly."`

> 			choice
> 			`	"Don't worry, you'll be surprised how fast my ship flies. We'll get there in no time."`
> 				goto notime
> 			`	"Yes, a lot of captains are quite busy these days."`
> 			`	"Yes, it seems that way nowadays," Sophie agrees. "Everyone feels more tense or uncertain, and there's always more to do and less time to do it."`

Reviewer 2: ```suggestion
			choice
				`	"Don't worry, you'll be surprised how fast my ship flies. We'll get there in no time."`
					goto notime
				`	"Yes, a lot of captains are quite busy these days."`
			`	"It seems that way," Sophie agrees. "Everyone feels more tense or uncertain, and there's always more to do and less time to do it."`
```

			`	She pauses for a moment, lost in her memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. He said that he would not make it easy, though; he was a great fan of fantasy books, and in those stories there is always something mysterious at play. Instead, he left behind a few systems that before we should visit before we go to Prime, where he left something behind for us.`
			`	"Of course, it would be much easier if we could go directly to Prime, but we want to respect our father's last will." She looks directly at you now, "And that is why we need your help. Could you help us again and take us through all these systems?"`

> 			choice
> 			`	"Yes, I'd be glad to."`
> 					goto accepted
> 			`	"I'm sorry, but that sounds too complicated to me."`

Reviewer 2: ```suggestion
			choice
				`	"Yes, I'd be glad to."`
						goto accepted
				`	"I'm sorry, but that sounds too complicated to me."`
```

	on enter "Gorvi"
		dialog `Sophie and Liam both talk about how much power a supernova must have, even an old one, if it leaves this much behind.`
	on enter "Enif"
		dialog `If there was ever a prime example of how much the human race still has to learn, it is the strange and impressive yet still unfinished Quarg ringworld in this system. Liam tells you, "Our father was a very intriguing man, and he always talked about the Quarg and their ringworlds and how it would be another thousand years before humanity could build anything similar. We never knew how big it was until now!"`
	on complete
		conversation
			`The siblings are uncharacteristically quiet, perhaps in anticipation of finally arriving at their destination. It is only when you've all stepped out into Prime's spaceport that Sophie says, "Captain <last>, you've helped us immensely, and we can't thank you enough for bringing us all across the galaxy! But we have one last request. Both of us think that you should come with us to see what our father left behind. Do you want to see it?"`
			choice
				`	"Now that we've come so far, sure."`
					goto continue
				`	"I'm sorry, I've got to be somewhere else."`

> 			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will! Oh, and here's your payment," Liam says, handing you credit chip of <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`
> 			payment 50000

Reviewer 2: ```suggestion
			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will! Oh, and here's your payment," Liam says, handing you credit chip of <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`
			action
				payment 50000
			decline
```

			decline
			label continue
			`	Liam pulls out a map, not the starmap he had before but a map of Prime. "Apparently our father bought a whole mountain on this planet, and he wants us to go there."`
			choice
				`	"Sounds good. I'll follow your lead."`
						goto shuttle
				`	"We could also have a look around here, and see why your father chose this planet."`
			`	"You're right. I hadn't thought of that," Sophie says."`

> 			`	All three of you walk around the spaceport district for some time. Coming into a giant plaza, you see one of the newest ships of Betelgeuse being displayed.

Reviewer 2: ```suggestion
			`	All three of you walk around the spaceport district for some time. Coming into a giant plaza, you see one of the newest ships of Betelgeuse being displayed.`
```

			`	Liam's face brightens as if he just understood something. "I think our father chose this planet because he was always a fan of Betelgeuse ships. If I recall correctly, he was a test pilot here, and once he was allowed to keep one of their experimental models as a gift for being one of their most productive pilots." Sophie nods, "Right! Now that you say that, I remember that too. Anyways, shall we go to the mountain now?"`
			choice

> 				`	"We should take a shuttle. It'll be the fastest way."
> 						goto shuttle
> 				`	"We have time. Let's take the maglev."

Reviewer 2: ```suggestion
				`	"We should take a shuttle. It'll be the fastest way."`
						goto shuttle
				`	"We have time. Let's take the maglev."`
```

			`			Your father."`
			`	Understanding the personal nature of the message, you leave the two of them alone for a bit, walking around and looking at the artifacts.`
			`	After some time, the siblings call you back to them. Sophie starts, "Captain <last>, Liam and I have thought through about this, and we think it would be a shame if we just kept this sword as a memory to him. It needs to be used. It was our father's one and only weapon, and it has been many places. Take it to many more." She pulls the sword from the pedestal and hands it to you. "It is yours now."`
			choice
				`	"I would be more than grateful to accept it."`
						goto accepted
				`	"This is too much for me to take."`
						goto declined
			label declined
			`	"Are you sure?" Liam asks. "This is a one time offer. It would be wasted on us."`
			choice
				`	"Yes, I'm sure."`
				`	"Well, if you frame it that way, I will take it."`
					goto accepted
			`	"I see. Please, at least take this extra payment of <credits>," Liam says. "It is for you traveling so much with us."`

> 				action
> 					payment 50000

Reviewer 2: ```suggestion
				action
					payment 50000
				decline
```
If you're finishing a conversation, you will need a decline endpoint node.

				goto end
			label accepted
			action
				outfit "Old Man's Sword" 1
			label end
			`	"Well, it seems like this is the end of our short journey together, Captain <last>," Sophie says.`
			`	"It was a pleasure to travel with you," adds Liam. "We will always thank you for helping us with our father's will. Here's your payment of <payment>, and farewell." `

> 			`	You say goodbye to them, and leave them alone in their father's dome.`
> 				payment 50000
> 				decline

Reviewer 2: ```suggestion
			`	You say goodbye to them, and leave them alone in their father's dome.`
				action
					payment 50000
				decline
```

			label helpornot
			choice
			`	"Of course I can."`
					goto agreed
			`	"I'm sorry, but I've got more urgent things to do. You'll need to find someone else."`
			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, with both heads looking down.`
				decline
			label agreed
			`    "Fantastic! My name is Sophie, and this is my younger brother," she says, pointing at the man to her side, who goes to shake your hand. "Hi, I'm Liam. We have been looking for a ride for some time now, we've already started worrying that we wouldn't get there in time! You'll have to bring us there quickly."`
			choice
			`	"Don't worry, you'll be surprised how fast my ship flies. We'll get there in no time."`
				goto notime
			`	"Yes, a lot of captains are quite busy these days."`
			`	"Yes, it seems that way nowadays," Sophie agrees. "Everyone feels more tense or uncertain, and there's always more to do and less time to do it."`
				goto ship
			label notime

> 			`	Liam's voice barely hides his excitement at finding an available captain. "Well, let's see that ship of yours then, and show us if it's as fast as you say it is!"

Reviewer 2: ```suggestion
			`	Liam's voice barely hides his excitement at finding an available captain. "Well, let's see that ship of yours then, and show us if it's as fast as you say it is!"`
```

			label ship
			`	You lead them to your ship, and help them with their bags as they board.`
				accept
	on visit
		dialog phrase "generic passenger on visit"
	on complete
		dialog `When you arrive at New China's spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
		payment 50000

mission "A dead man's path II"
	name "Following Footprints"
	description "Bring Sophie and Liam through <waypoints>, before finally bringing them to <destination>."
	passengers 2
	source
		planet "New China"
	destination
		planet "Prime"
	waypoint "Terminus"
	waypoint "Aspidiske"
	waypoint "Gorvi"
	waypoint "Enif"
	to offer
		has "A dead man's path I: done"
	on offer
		conversation

> 			`	After about two hours they come back, although unexpectedly they both have broad smiles on their face. You flag them down from your table, and they eagerly take a seat. 

Reviewer 2: ```suggestion
			`	After about two hours they come back, although unexpectedly they both have broad smiles on their face. You flag them down from your table, and they eagerly take a seat.`
```

	on complete
		conversation
			`The siblings are uncharacteristically quiet, perhaps in anticipation of finally arriving at their destination. It is only when you've all stepped out into Prime's spaceport that Sophie says, "Captain <last>, you've helped us immensely, and we can't thank you enough for bringing us all across the galaxy! But we have one last request. Both of us think that you should come with us to see what our father left behind. Do you want to see it?"`
			choice
				`	"Now that we've come so far, sure."`
					goto continue
				`	"I'm sorry, I've got to be somewhere else."`
			action
				payment 50000
			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will! Oh, and here's your payment," Liam says, handing you a credit chip for <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`

> 			decline

Author: ```suggestion
				decline
```

			`	For a long while, the three of you continue through the stone hallway, the light following your movements, until it opens up into a dome-shaped room, brilliantly bathed in blue light. On the walls, there are all kinds of alien artifacts lined up next to each other, but at the center of the room there is a pedestal with a sword inserted into it. A small tablet stands in a socket in front of it.`
			choice
				`	"(Go up to the sword.)"`
						goto sword-scene
				`	"(Take a look around first.)"`
			`	You walk around, looking at the various alien artifacts. "Wow, he must have always brought us back the small items from his expeditions, because these artifacts over here are huge!" Liam exclaims from another part of the dome. "How did he even move all these here?"`
			`	Now that Liam points it out, many of the larger artifacts look similar to the Sheragi technology you studied with Albert Foster.`
				to display
					has "Sheragi Archaeology: The Emerald Sword 3: done"
			`	After some time, Sophie calls you both to come to look at the tablet.`
			label sword-scene
			scene "scene/sword dome"
			`	Approaching the tablet, there is a short message written on it:`

> 			`	"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
> 			`	But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
> 			`	This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
> 			`	Your father."`
> 			`	You leave the two of them alone for a bit, walking around and looking at the artifacts.`
> 			`	After some time they call you to them. Sophie starts, "Captain <last>, Liam and I have thought through about this, and we think it would be a shame if we just kept this sword as a memory to him. It needs to be used. It was our father's one and only weapon, and it has been many places. Take it to many more." She pulls the sword from the pedestal and hands it to you. "It is yours now."`

Reviewer 2: ```suggestion
			`			"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
			`			But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
			`			This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
			`			Your father."`
			`	Understanding the personal nature of the message, you leave the two of them alone for a bit, walking around and looking at the artifacts.`
			`	After some time, the siblings call you back to them. Sophie starts, "Captain <last>, Liam and I have thought through about this, and we think it would be a shame if we just kept this sword as a memory to him. It needs to be used. It was our father's one and only weapon, and it has been many places. Take it to many more." She pulls the sword from the pedestal and hands it to you. "It is yours now."`
```
I'm going to separate it a bit from the rest of the dialog by adding more indents.

			choice
				`	"I would be more than grateful to accept it."`
						goto accepted
				`	"This is too much for me to take."`
						goto declined
			label declined
			`	"Are you sure?" Liam asks. "This is a one time offer. It would be wasted on us."`
			choice
				`	"Yes, I'm sure."`
				`	"Well, if you frame it that way, I will take it."`
					goto accepted
			`	"I see. Please, at least take this extra payment of <credits>," Liam says. "It is for you traveling so much with us."`
				action
					payment 50000
				goto end

> 			label accept-sword
> 			`	"Please take it on some more adventures," Sophie says.`

Reviewer 2: ```suggestion
			label accepted
```
I have to remove this line because with the new dialogue, Sophie repeats herself now. I might add a new piece of dialogue here for the player to try out the sword like it had before.

			action
				outfit "Old Man's Sword" 1
			label end
			`	"Well, it seems like this is the end of our short journey together, Captain <last>," Sophie says.`
			`	"It was a pleasure to travel with you," adds Liam. "We will always thank you for helping us with our father's will. Here's your payment of <payment>, and farewell." `

> 			`	You say goodbye to them and leave them alone in the dome.`

Reviewer 2: ```suggestion
			`	You say goodbye to them, and leave them alone in their father's dome.`
```

			`	"Of course, it would be much easier if we could go directly to Prime, but we want to respect our father's last will." She looks directly at you now, "And that is why we need your help. Could you help us again and take us through all these systems?"`
			choice
				`	"Yes, I'd be glad to."`
						goto accepted
				`	"I'm sorry, but that sounds too complicated to me."`
			`	"Well, it looks like we'll need to find somebody else," Sophie says. They both thank you for bringing them this far and walk away.`
				decline
			label accepted
			`	Liam pulls a starmap out of his bag. "These are the systems our father wants us to visit. Apparently they all have something unique to them. We weren't expecting this trip so we'll need to buy some supplies, but as soon as you're ready we'll be on board."`
				accept
	on enter "Terminus"

> 		dialog `It's often thought that there are nothing to these uninhabited by the Deep, but Sophie points out that there's a strange anomaly in this system.`

Reviewer 2: ```suggestion
		dialog `It's often thought that there are nothing to these uninhabited stars by the Deep, but Sophie points out that there's a strange anomaly in this system.`
```

	on complete
		conversation
			`The siblings are uncharacteristically quiet, perhaps in anticipation of finally arriving at their destination. It is only when you've all stepped out into Prime's spaceport that Sophie says, "Captain <last>, you've helped us immensely, and we can't thank you enough for bringing us all across the galaxy! But we have one last request. Both of us think that you should come with us to see what our father left behind. Do you want to see it?"`
			choice
				`	"Now that we've come so far, sure."`
					goto continue
				`	"I'm sorry, I've got to be somewhere else."`
			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will! Oh, and here's your payment," Liam says, handing you a credit chip for <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`
			action
				payment 50000

> 			decline

Author: ```suggestion
			``
				decline
```

> 			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will! Oh, and here's your payment," Liam says, handing you a credit chip for <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`
> 			action
> 				payment 50000
> 			decline
> 			decline

Reviewer 2: ```suggestion
			action
				payment 50000
			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will! Oh, and here's your payment," Liam says, handing you a credit chip for <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`
			decline
```
I didn't know that about the backticks. Sorry RisingLeaf!

			`	"Well, now that you say it, I recently bought a small camera drone," Liam responds. "We can use it to scout out the hallway. It's only in infrared, so we won't see too many details," he adds. Liam starts the drone and guides it using a small remote control with integrated display. After some time, the drone flies back out and he tells you, "I think it is safe. We can go."`
			label enter
			`	The three of you enter the hallway behind the opening in single file. Strangely, the stone walls are perfectly smoothed off, and the light that guides you through seems to come out of nowhere. "To use alien technology for something like this," Sophie laughs, "That's totally our dad."`
			`	For a long while, the three of you continue through the stone hallway, the light following your movements, until it opens up into a dome-shaped room, brilliantly bathed in blue light. On the walls, there are all kinds of alien artifacts lined up next to each other, but at the center of the room there is a pedestal with a sword inserted into it. A small tablet stands in a socket in front of it.`
			choice
				`	"(Go up to the sword.)"`
						goto sword-scene
				`	"(Take a look around first.)"`

> 			`	You walk around, looking at the various alien artifacts. "Wow, he must have always brought us back the small items from his expeditions, because these artifacts over here are huge!" Liam exclaims from another part of the dome. "How did he even move all these here?"`

Reviewer 2: ```suggestion
			`	You walk around to the right of the dome to a collection of various alien artifacts, all steel in color and rather blocky. "Wow, he must have always brought us back the small items from his expeditions, because these artifacts over here are huge!" Liam exclaims from another part of the dome. "How did he even move all these here?"`
```

> 			`	Now that Liam points it out, many of the larger artifacts look similar to the Sheragi technology you studied with Albert Foster.`

Reviewer 2: ```suggestion
			`	Now that Liam points it out, many of the larger artifacts over his way look similar to the Sheragi technology you studied with Albert Foster.`
```

			`	After some time, Sophie calls you both to come to look at the tablet.`
			label sword-scene
			scene "scene/sword dome"
			`	Approaching the tablet, there is a short message written on it:`
			`			"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
			`			But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
			`			This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
			`			Your father."`

> 			`	Understanding the personal nature of the message, you leave the two of them alone for a bit, walking around and looking at the artifacts.`

Reviewer 2: ```suggestion
			`	Understanding the personal nature of the message, you leave the two of them alone for a bit, walking around and looking at the artifacts on the left of the dome.`
```

			`	After some time, the siblings call you back to them. Sophie starts, "Captain <last>, Liam and I have thought through about this, and we think it would be a shame if we just kept this sword as a memory to him. It needs to be used. It was our father's one and only weapon, and it has been many places. Take it to many more." She pulls the sword from the pedestal and hands it to you. "It is yours now."`
			choice
				`	"I would be more than grateful to accept it."`
						goto accepted
				`	"This is too much for me to take."`
						goto declined
			label declined
			`	"Are you sure?" Liam asks. "This is a one time offer. It would be wasted on us."`
			choice
				`	"Yes, I'm sure."`
				`	"Well, if you frame it that way, I will take it."`
					goto accepted

> 			`	"I see. Please, at least take this extra payment of <credits>," Liam says. "It is for you traveling so much with us."`

Reviewer 2: ```suggestion
			`	"I see. Please, at least take an extra payment of <credits>," Liam says. "It is for you traveling so much with us."`
```

> 			`	"I see. Please, at least take this extra payment of <credits>," Liam says. "It is for you traveling so much with us."`
> 			action
> 				payment 50000

Reviewer 2: ```suggestion
			action
				payment 50000
			`	"I see. Please, at least take this extra payment of <credits>," Liam says. "It is for you traveling so much with us."`
```

			label accepted
			action
				outfit "Old Man's Sword" 1

> 			label end
> 			`	"Well, it seems like this is the end of our short journey together, Captain <last>," Sophie says.`
> 			`	"It was a pleasure to travel with you," adds Liam. "We will always thank you for helping us with our father's will. Here's your payment of <payment>, and farewell." `
> 			`	You say goodbye to them, and leave them alone in their father's dome.`
> 			action
> 				payment 50000
> 			decline

Reviewer 2: ```suggestion
			label end
			action
				payment 50000
			`	"Well, it seems like this is the end of our short journey together, Captain <last>," Sophie says.`
			`	"It was a pleasure to travel with you," adds Liam. "We will always thank you for helping us with our father's will, we never imagined that we would come so far and we would have never done it without you. It means so much to us. We fare you well among the stars!"`
			`	After he hands you a credit ship of <payment>, you say goodbye to them and leave them in their father's dome.`
			decline
```

> 				decline

Reviewer 2: ```suggestion
			decline
```
Please green tick!

			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, with both heads looking down.`
				decline
			label agreed
			`    "Fantastic! My name is Sophie, and this is my younger brother," she says, pointing at the man to her side, who goes to shake your hand. "Hi, I'm Liam. We have been looking for a ride for some time now, we've already started worrying that we wouldn't get there in time! You'll have to bring us there quickly."`
			choice
				`	"Don't worry, you'll be surprised how fast my ship flies. We'll get there in no time."`
					goto notime
				`	"Yes, a lot of captains are quite busy these days."`
			`	"It seems that way," Sophie agrees. "Everyone feels more tense or uncertain, and there's always more to do and less time to do it."`
				goto ship
			label notime
			`	Liam's voice barely hides his excitement at finding an available captain. "Well, let's see that ship of yours then, and show us if it's as fast as you say it is!"`
			label ship
			`	You lead them to your ship, and help them with their bags as they board.`
				accept
	on visit
		dialog phrase "generic passenger on visit"
	on complete

> 		log "A last will" `Dropped of two siblings on New China to be told their father's last will. They might need another ride.`

Reviewer 2: ```suggestion
		log `Helped a pair of siblings named Sophie and Liam by bringing them to New China so they can hear their father's last will.`
```
This is more in line with other logs. Also, this happens: 
![Screenshot_20221219_112202](https://user-images.githubusercontent.com/115441627/208322488-50e3b08e-5eaa-41da-97ff-7ba395575355.png)

Author: This should not happen

		dialog `When you arrive at New China's spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
		payment 50000

mission "A dead man's path II"
	name "Following Footprints"

> 	description "Bring Sophie and Liam through <waypoints>, before finally bringing them to <destination>."

Reviewer 2: ```suggestion
	description "Bring Sophie and Liam through <waypoints>, before finally bringing them to <destination>. They will pay you another <payment>."
```

	passengers 2
	source
		planet "New China"
	destination
		planet "Prime"
	waypoint "Terminus"
	waypoint "Aspidiske"
	waypoint "Gorvi"
	waypoint "Enif"
	to offer
		has "A dead man's path I: done"
	on offer
		conversation
			`	After about two hours they come back, although unexpectedly they both have broad smiles on their face. You flag them down from your table, and they eagerly take a seat.`
			`	Liam speaks as soon as he can. "Captain <last>, it turns out we will need a bit more than just a ride home! Have you decided to help us again?"`
			choice
				`	"But what do you want me to do?"`
				`	"Sorry, but I'd be stupid to accept a mission I don't know anything about."`

> 			`	Sophie looks at her brother amusedly. "Of course, my brother just can't wait to get started," She turns back to you. "Maybe I should start with who our father was. We do not know much about him, he came home only occasionally. Don't get me wrong, our parents weren't fighting, but his passion just did not allow him to stay in one place for very long. He was a captain, not unlike you, and he was restless to explore even the furthest parts of the galaxy. We believe he traveled further than most human captains; sometimes he brought small items back home that nobody could explain."`

Reviewer 2: ```suggestion
			`	Sophie looks at her brother amusedly. "Of course, my brother just can't wait to get started!" She turns back to you. "Maybe I should start with who our father was. We don't didn't get to know him very well, he came home only occasionally. Don't get me wrong, our parents weren't fighting, but his passion just did not allow him to stay in one place for very long. He was a captain, not unlike you, and he was restless to explore even the furthest parts of the galaxy. We believe he traveled further than most human captains; sometimes he brought small items back home that nobody could explain."`
```

> 			`	She pauses for a moment, lost in her memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. He said that he would not make it easy, though; he was a great fan of fantasy books, and in those stories there is always something mysterious at play. Instead, he left behind a few systems that before we should visit before we go to Prime, where he left something behind for us.`

Reviewer 2: ```suggestion
			`	She pauses for a moment, lost in her memories. "Anyways," she continues, "in his testament he told us that he left something behind for us on Prime. He said that he would not make it easy, though; he was a great fan of fantasy books, and in those stories there is always something mysterious at play that we do not understand the meaning of until the very end. Instead, he told us of a few systems that before we should visit before we go to Prime.`
```

			`	"Of course, it would be much easier if we could go directly to Prime, but we want to respect our father's last will." She looks directly at you now, "And that is why we need your help. Could you help us again and take us through all these systems?"`
			choice
				`	"Yes, I'd be glad to."`
						goto accepted
				`	"I'm sorry, but that sounds too complicated to me."`
			`	"Well, it looks like we'll need to find somebody else," Sophie says. They both thank you for bringing them this far and walk away.`
				decline
			label accepted
			`	Liam pulls a starmap out of his bag. "These are the systems our father wants us to visit. Apparently they all have something unique to them. We weren't expecting this trip so we'll need to buy some supplies, but as soon as you're ready we'll be on board."`
				accept
	on enter "Terminus"
		dialog `It's often thought that there are nothing to these uninhabited stars by the Deep, but Sophie points out that there's a strange anomaly in this system.`
	on enter "Aspidiske"

> 		dialog `While Sophie and Liam are both quite chatty at times, the beauty of this star stops all of their conversations.`

Reviewer 2: ```suggestion
		dialog `While Sophie and Liam are both quite chatty at times, the beauty of this star stops all their conversations.`
```

	on enter "Gorvi"

> 		dialog `Sophie and Liam both talk about how much power a supernova must have, even an old one, if it leaves this much behind.`

Reviewer 2: ```suggestion
		dialog `Sophie and Liam both go on about how much power a supernova must have, even an old one, if it leaves this much behind.`
```

	on enter "Enif"

> 		dialog `If there was ever a prime example of how much the human race still has to learn, it is the strange and impressive yet still unfinished Quarg ringworld in this system. Liam tells you, "Our father was a very intriguing man, and he always talked about the Quarg and their ringworlds and how it would be another thousand years before humanity could build anything similar. We never knew how big it was until now!"`

Reviewer 2: ```suggestion
		dialog `If there was ever a prime example of how much the human race still has to learn, it is the strange and impressive yet still unfinished Quarg ringworld in this system. Liam tells you, "Our father was a very curious man, and he always talked about the Quarg and their ringworlds and how it would be another thousand years before humanity could build anything similar. We never knew how big it was until now!"`
```

> 	on complete

Reviewer 2: ```suggestion
	on visit
		dialog phrase "generic missing waypoint or passengers"
	on complete
```

> 		log "A last will" `Brought the siblings to various locations, before finally bringing them to Prime, where their father has left something for them.`

Reviewer 2: ```suggestion
		log `Brought Sophie and Liam around the galaxy to various locations that their dead father asked them to visit, before finally bringing them to Prime where he left something for them.`
```

						goto enter
				`	"We don't know what's inside there. Be careful."`
			`	"Well, now that you say it, I recently bought a small camera drone," Liam responds. "We can use it to scout out the hallway. It's only in infrared, so we won't see too many details," he adds. Liam starts the drone and guides it using a small remote control with integrated display. After some time, the drone flies back out and he tells you, "I think it is safe. We can go."`
			label enter
			`	The three of you enter the hallway behind the opening in single file. Strangely, the stone walls are perfectly smoothed off, and the light that guides you through seems to come out of nowhere. "To use alien technology for something like this," Sophie laughs, "That's totally our dad."`
			`	For a long while, the three of you continue through the stone hallway, the light following your movements, until it opens up into a dome-shaped room, brilliantly bathed in blue light. On the walls, there are all kinds of alien artifacts lined up next to each other, but at the center of the room there is a pedestal with a sword inserted into it. A small tablet stands in a socket in front of it.`
			choice

> 				`	"(Go up to the sword.)"`
> 						goto sword-scene
> 				`	"(Take a look around first.)"`

Reviewer 2: ```suggestion
				`	(Go up to the sword.)`
						goto sword-scene
				`	(Take a look around first.)`
```

			`			Your father."`
			`	Understanding the personal nature of the message, you leave the two of them alone for a bit, walking around and looking at the artifacts on the left of the dome.`
			`	After some time, the siblings call you back to them. Sophie starts, "Captain <last>, Liam and I have thought through about this, and we think it would be a shame if we just kept this sword as a memory to him. It needs to be used. It was our father's one and only weapon, and it has been many places. Take it to many more." She pulls the sword from the pedestal and hands it to you. "It is yours now."`
			choice
				`	"I would be more than grateful to accept it."`
						goto accepted
				`	"This is too much for me to take."`
						goto declined
			label declined
			`	"Are you sure?" Liam asks. "This is a one time offer. It would be wasted on us."`
			choice
				`	"Yes, I'm sure."`
				`	"Well, if you frame it that way, I will take it."`
					goto accepted
			action
				payment 50000

> 			`	"I see. Please, at least take this extra payment of <credits>," Liam says. "It is for you traveling so much with us."`

Reviewer 2: ```suggestion
			`	"I see. Please, at least take this extra payment of <payment>," Liam says. "It is for you traveling so much with us."`
```

> 			`	"I see. Please, at least take this extra payment of <payment>," Liam says. "It is for you traveling so much with us."`

Reviewer 2: ```suggestion
			action
				set "A Dead Man's Path: did not accept old man's sword"
			`	"I see. Please, at least take this extra payment of <payment>," Liam says. "It is for you traveling so much with us."`
				goto end
```
Whoops. Otherwise you get the sword either way. I need to add an extra bit to show that Liam gives you 100k credits instead of 50k twice, it's bugging me.

			label accepted
			action
				outfit "Old Man's Sword" 1
			label end
			action
				payment 50000
			`	"Well, it seems like this is the end of our short journey together, Captain <last>," Sophie says.`
			`	"It was a pleasure to travel with you," adds Liam. "We will always thank you for helping us with our father's will, we never imagined that we would come so far and we would have never done it without you. It means so much to us. We fare you well among the stars!"`

> 			`	After he hands you a credit ship of <payment>, you say goodbye to them and leave them in their father's dome.`
> 			decline

Reviewer 2: ```suggestion
			`	After he hands you a credit ship for <payment>, you say goodbye to them and leave them in their father's dome.`
				decline
```
Strangely this decline was shown in the conversation screen. Here's a screenshot: 
![Screenshot_20221219_111712](https://user-images.githubusercontent.com/115441627/208322301-285bccb2-c582-43e1-b795-bb7a0608780a.png)
Maybe the decline isn't needed.

Author: hmmm

> 			`	After he hands you a credit ship for <payment>, you say goodbye to them and leave them in their father's dome.`

Reviewer 2: ```suggestion
			`	After he hands you a credit chip of <payment>, you say goodbye to them and leave them in their father's mysterious dome.`
			to display
				not "A Dead Man's Path: did not accept old man's sword"
			`	After he hands you two credit chips of <payment>, you say goodbye to them and leave them in their father's mysterious dome.`
			to display
				has "A Dead Man's Path: did not accept old man's sword"
```
Hopefully this works.

			`			Your father."`
			`	Understanding the personal nature of the message, you leave the two of them alone for a bit, walking around and looking at the artifacts.`
			`	After some time, the siblings call you back to them. Sophie starts, "Captain <last>, Liam and I have thought through about this, and we think it would be a shame if we just kept this sword as a memory to him. It needs to be used. It was our father's one and only weapon, and it has been many places. Take it to many more." She pulls the sword from the pedestal and hands it to you. "It is yours now."`
			choice
				`	"I would be more than grateful to accept it."`
						goto accepted
				`	"This is too much for me to take."`
						goto declined
			label declined
			`	"Are you sure?" Liam asks. "This is a one time offer. It would be wasted on us."`
			choice
				`	"Yes, I'm sure."`
				`	"Well, if you frame it that way, I will take it."`
					goto accepted
			`	"I see. Please, at least take this extra payment of <credits>," Liam says. "It is for you traveling so much with us."`
			action
				payment 50000

> 			``
> 				decline

Reviewer 2: ```suggestion
```
Sorry, it doesn't need that decline that I put there. I was just about to suggest those changes you just committed too.

			label accepted
			action
				outfit "Old Man's Sword" 1
			label end
			`	"Well, it seems like this is the end of our short journey together, Captain <last>," Sophie says.`
			`	"It was a pleasure to travel with you," adds Liam. "We will always thank you for helping us with our father's will. Here's your payment of <payment>, and farewell." `
			`	You say goodbye to them, and leave them alone in their father's dome.`
			action
				payment 50000

> 			``

Reviewer 2: ```suggestion
```
These backticks as well?

				`	"Now that we've come so far, sure."`
					goto continue
				`	"I'm sorry, I've got to be somewhere else."`
			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will! Oh, and here's your payment," Liam says, handing you a credit chip for <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`
			action
				payment 50000

> 			decline

Reviewer: double decline here and this doesnt work anyway, decline to be on a text element
Either put the action before the text or have this:
Empty text:
``
decline (indented ofc)

			`	You say your goodbyes, leaving to your ship and returning to the stars.`
				decline

> mission "Wool Smuggling 1"

Author: ```suggestion

mission "Wool Smuggling 1"
```

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

> mission "A dead man's path I"

Reviewer 4: ```suggestion
mission "A Dead Man's Path 1"
```

			choice
				`	"Don't worry, you'll be surprised how fast my ship flies. We'll get there in no time."`
					goto notime
				`	"Yes, a lot of captains are quite busy these days."`
			`	"It seems that way," Sophie agrees. "Everyone feels more tense or uncertain, and there's always more to do and less time to do it."`
				goto ship
			label notime
			`	Liam's voice barely hides his excitement at finding an available captain. "Well, let's see that ship of yours then, and show us if it's as fast as you say it is!"`
			label ship
			`	You lead them to your ship, and help them with their bags as they board.`
				accept
	on visit
		dialog phrase "generic passenger on visit"
	on complete
		log `Helped a pair of siblings named Sophie and Liam by bringing them to New China so they can hear their father's last will.`
		dialog `When you arrive at New China's spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
		payment 50000

> mission "A dead man's path II"

Reviewer 4: ```suggestion
mission "A Dead Man's Path 2"
```

	name "Following Footprints"
	description "Bring Sophie and Liam through <waypoints>, before finally bringing them to <destination>. They will pay you another 50,000 credits."
	passengers 2
	source
		planet "New China"
	destination
		planet "Prime"
	waypoint "Terminus"
	waypoint "Aspidiske"
	waypoint "Gorvi"
	waypoint "Enif"
	to offer
		has "A dead man's path I: done"
	on offer
		conversation

> 			`	After about two hours they come back, although unexpectedly they both have broad smiles on their face. You flag them down from your table, and they eagerly take a seat.`

Reviewer 2: ```suggestion
			`After waiting two hours in the spaceport's bar, Liam and Sophie come back with unexpectedly broad smiles on their faces. You flag them down from your table, and they eagerly take a seat.`
```
I just realised that this wouldn't make sense if you came back to New China later, and I removed the spaceport bar part so I'll have to add it back.

	on complete
		log `Brought Sophie and Liam around the galaxy to various locations that their dead father asked them to visit, before finally bringing them to Prime where he left something for them.`
		conversation
			`The siblings are uncharacteristically quiet, perhaps in anticipation of finally arriving at their destination. It is only when you've all stepped out into Prime's spaceport that Sophie says, "Captain <last>, you've helped us immensely, and we can't thank you enough for bringing us all across the galaxy! But we have one last request. Both of us think that you should come with us to see what our father left behind. Do you want to see it?"`
			choice
				`	"Now that we've come so far, sure."`
					goto continue
				`	"I'm sorry, I've got to be somewhere else."`
			action
				payment 50000
			`	"Well, that's okay. We'll always appreciate the help you've given us in fulfilling our father's will! Oh, and here's your payment," Liam says, handing you a credit chip for <payment>. "We wish you well through the stars!" Again, the siblings both thank you profusely for the trip, and you watch them leave Prime's spaceport with their heads up.`
				decline
			label continue

> 			`	Liam pulls out a map, not the starmap he had before but a map of Prime. "Apparently our father bought a whole mountain on this planet, and he wants us to go there."`

Reviewer 4: ```suggestion
			`	Liam pulls out a map, not the star map he had before but a map of Prime. "Apparently our father bought a whole mountain on this planet, and he wants us to go there."`
```

			choice
				`	(Go up to the sword.)`
						goto sword-scene
				`	(Take a look around first.)`
			`	You walk around to the right of the dome to the collection of alien objects, all steel in color and rather blocky. "Wow, he must have always brought us back the small items from his expeditions, because these artifacts over here are huge!" Liam exclaims from another part of the dome. "How did he even move all these here?"`
			`	Now that Liam points it out, many of the larger artifacts over his way look similar to the Sheragi technology you studied with Albert Foster.`
				to display
					has "Sheragi Archaeology: The Emerald Sword 3: done"
			`	After some time, Sophie calls you both to come to look at the tablet.`
			label sword-scene
			scene "scene/sword dome"
			`	Approaching the tablet, there is a short message written on it:`

> 			`			"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
> 			`			But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
> 			`			This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
> 			`			Your father."`

Reviewer 4: ```suggestion
			`			"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
			`			"But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
			`			"This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
			`			"Your father."`
```
Quotes at the beginning (but not at the end if it still continues) of a paragraph that's continuing previous dialog is standard.

			`	"This sword was always at our father's side, and it has been many places. Take it to many more." She pulls the sword from the pedestal, and gives it one last look over before handing over it to you. "It is yours now."`
			choice
				`	"I would be more than grateful to accept it."`
						goto accepted
				`	"This is too much for me to take."`
						goto lastchance
			label lastchance
			`	"Are you sure?" Sophie asks. "This is a one time offer. It would be wasted on us."`
			choice
				`	"Yes, I'm sure."`
					goto declined
				`	"Well, if you frame it that way, I will take it."`
			label accepted
			action
				outfit "Old Man's Sword" 1
				payment 50000
			`	"Well, we'll need some time to take it all in and arrange things, find a place to stay on Prime and all that," Sophie says, "but we won't keep you from your ship any longer. It was a pleasure to travel with you, Captain <last>."`

> 			`	Liam adds, "We will always thank you for helping us with our father's will, we never imagined that we would come so far and we would have never done it without you. It means so much to us. We wish you well through the stars, Captain <last>."`

Reviewer 5: ```suggestion
			`	Liam adds, "We will always thank you for helping us with our father's will. We never imagined that we would come so far and we would have never done it without you. It means so much to us. We wish you well through the stars, Captain <last>."`
```

> 			`	After Liam hands you a credit chip of <payment>, you say goodbye to the two siblings and leave them in their father's mysterious dome.`

Reviewer 4: ```suggestion
			`	After Liam hands you a credit chip of 50,000 credits, you say goodbye to the two siblings and leave them in their father's mysterious dome.`
```
Mentioned below how a conversation can't properly update the <payment> substitution depending on actions. Seems like it'll need to be hardcoded.

				decline
			label declined
			action
				payment 100000
			`	"I see... please, if you won't take our father's sword, take an extra payment for having to put up with us all the way up until now." She takes the sword away and inserts it back into the pedestal.`
			`	"Well, we'll need some time to take it all in and arrange things, find a place to stay on Prime and all that," Sophie says, "but we won't keep you from your ship any longer. It was a pleasure to travel with you, Captain <last>."`

> 			`	Liam adds, "We will always thank you for helping us with our father's will, we never imagined that we would come so far and we would have never done it without you. It means so much to us. We wish you well through the stars, Captain <last>."`

Reviewer 4: ```suggestion
			`	Liam adds, "We will always thank you for helping us with our father's will. We never imagined that we would come so far and we would have never done it without you. It means so much to us. We wish you well through the stars, Captain <last>."`
```

> 			`	After Liam hands you a few credit chips for <payment>, you say goodbye to the two siblings and leave them in their father's mysterious dome.`

Reviewer 5: This is identical to the paragraph above. You can just delete this and replace it with a goto message that will direct to the paragraph above.

Reviewer 4: ```suggestion
			`	After Liam hands you a few credit chips for 100,000 credits, you say goodbye to the two siblings and leave them in their father's mysterious dome.`
```
On top of the <payment> having a difference between "a credit chip"/"a few credit chips" making them different, we'll also probably need to hard-write the payments since the substitution can't detect them being updated on the fly.

Reviewer 2: I remember when I was testing if this worked that it did update, but sure, hard-writing it is fine.

				`	"Well, if you frame it that way, I will take it."`
					goto accepted
			action
				payment 50000
			action
				set "A Dead Man's Path: did not accept old man's sword"
			`	"I see. Please, at least take this extra payment of <payment>," Liam says. "It is for you traveling so much with us."`
				goto end
			label accepted
			action
				outfit "Old Man's Sword" 1
			label end
			action
				payment 50000
			`	"Well, it seems like this is the end of our short journey together, Captain <last>," Sophie says.`
			`	"It was a pleasure to travel with you," adds Liam. "We will always thank you for helping us with our father's will, we never imagined that we would come so far and we would have never done it without you. It means so much to us. We fare you well among the stars!"`
			`	After he hands you a credit chip of <payment>, you say goodbye to them and leave them in their father's mysterious dome.`
			to display
				not "A Dead Man's Path: did not accept old man's sword"
			`	After he hands you two credit chips of <payment>, you say goodbye to them and leave them in their father's mysterious dome.`
			to display
				has "A Dead Man's Path: did not accept old man's sword"

> 				decline

Author: ```suggestion
				decline

```
